docs/style-guide-for-guide-articles.md

```diff
@@ -137,7 +137,7 @@ To order your list, precede each line with a number.
 
 We do not recommend adding images to an article unless absolutely necessary. Images hurt Web Accessibility, they are difficult to maintain and may get outdated. They are also slow to load on poor connections, thus, hurt web performance.
 
-For including images in an article under compelling reasons, if they aren't already hosted somewhere else on the web, you will need to put them online yourself using a platform like [Imgur](https://imgur.com/) or [Flickr](https://www.flickr.com). You can also host images by committing them to a git repository and pushing it to GitHub. Then you can right-click the image and copy its URL.
+For including images in an article under compelling reasons, if they aren't already hosted somewhere else on the web, you will need to put them online yourself using a platform like [Imgur](https://cdn-media-1.freecodecamp.org/imgr/) or [Flickr](https://www.flickr.com). You can also host images by committing them to a git repository and pushing it to GitHub. Then you can right-click the image and copy its URL.
```

Comment:
@raisedadead We'll need to see what to do with this. We obviously cannot change it since users cannot upload to fCC's CDN afaik

Reply:
Good catch!

Reply:
I believe the changes are accidentally made with search and replace @ahmadabdolsaheb?

Can you double check all these again.

Reply:
@thecodingaviator Nice catch.
@raisedadead yes some of them were find and replace. I will look into them.

Reply:
Probably we need to change these instructions in the future to prevent campers from uploading to imgur all together.

Reply:
@ahmadabdolsaheb We'll need to discuss that. Please open an issue